core/bcast/bcast_test.go

```diff
@@ -127,6 +127,31 @@ func TestBroadcastBlindedBeaconBlock(t *testing.T) {
 	<-ctx.Done()
 }
 
+func TestValidatorRegistration(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	mock, err := beaconmock.New()
+	require.NoError(t, err)
+
+	registration := testutil.RandomCoreVersionedSignedValidatorRegistration(t).VersionedSignedValidatorRegistration
+
+	aggData := core.VersionedSignedValidatorRegistration{VersionedSignedValidatorRegistration: registration}
+
+	mock.SubmitValidatorRegistrationsFunc = func(ctx context.Context, registrations []*eth2api.VersionedSignedValidatorRegistration) error {
+		require.Equal(t, aggData.VersionedSignedValidatorRegistration, *registrations[0])
```

Comment:
this bcast is submitting 1 validator registration so we compare registrations[0]

should we be constructing an array with mutliple registrations and checking that array is equal to the registrations (array) in the mocksSubmitValidatorRegistrationsFunc

Reply:
can do that or just check the length is 1